image.yaml

```diff
@@ -55,11 +55,11 @@ modules:
           - name: cct_module
             git:
                   url: https://github.com/jboss-openshift/cct_module.git
-                  ref: sprint-27
+                  ref: sprint-28
           - name: jboss-eap-modules
             git:
                   url: https://github.com/jboss-container-images/jboss-eap-modules.git
-                  ref: EAP72-CR5
+                  ref: EAP_721
```

Comment:
this should be sprint-31 : https://github.com/jboss-container-images/jboss-eap-7-openshift-image/blob/eap72-dev/cp-overrides.yaml#L12